BreakOut/spinn_breakout/visualiser/visualiser.py

```diff
@@ -68,23 +69,28 @@ def __init__(self, udp_port, key_input_connection,
                                       cmap=cmap, vmin=0.0, vmax=1.0)
 
         # Draw score using textbox
-        self.score_text = self.axis.text(0.5, 1.0, "0", color="green",
+        self.score_text = self.axis.text(0.5, 1.0, "0", color=BRIGHT_GREEN,
                                          transform=self.axis.transAxes,
                                          horizontalalignment="right",
                                          verticalalignment="top")
         # Hook key listeners
         self.fig.canvas.mpl_connect("key_press_event", self._on_key_press)
         self.fig.canvas.mpl_connect("key_release_event", self._on_key_release)
+        # Hide grid
+        self.axis.grid(False)
+        self.axis.set_xticklabels([])
+        self.axis.set_yticklabels([])
+        self.axis.axes.get_xaxis().set_visible(False)
 
     # ------------------------------------------------------------------------
     # Public methods
     # ------------------------------------------------------------------------
     def show(self):
         # Play animation
         self.animation = animation.FuncAnimation(self.fig, self._update,
-                                                 interval=20.0, blit=True)
+                                                 interval=20.0, blit=False)
```

Comment:
On my computer I am getting KeyErrors with blitting enabled. Maybe its an issue with my version of matplotlib (1.5.1). Also, my computer seems fine dealing with it. Will try it on 1.5.3 and report back